agent/tool-scripts/datalog/openshift-metrics-datalog

```diff
@@ -18,27 +18,17 @@ while True:
   r = requests.get('https://%s:8443/metrics' %(master), cert=('/run/pbench/keys/admin.crt', '/run/pbench/keys/admin.key'), verify=False)
   for values in [ line.strip() for line in r.text.splitlines()]:
     if '#' not in values[0]:
-      if 'apiserver_request_latencies' in values:
-        data = re.sub(r"{[^{}]+}", lambda x: x.group(0).replace(",",""), values)
-        name = data.split(' ')[0]
-        value = data.split(' ')[1]
-        print '{0},{1},{2}'.format(time_stamp, name, value)     
-      if 'apiserver_request_count' in values:
-        search_result = counter_regexp.search(values)
-        if search_result:
-          code = search_result.group(1)
-          resource = search_result.group(2)
-          verb = search_result.group(3)
-          counter = search_result.group(4)
-          key = resource + "_" + verb + "_" + str(code)
-          metric_nam = values.split('{')[0]
-          metric_name = metric_nam + "{resource=%s verb=%s code=%s}" %(resource,verb,code)
-          metric_value = values.split('}')[1]
-          print '{0},{1},{2}'.format(time_stamp, metric_name, metric_value)
-      if '{' in values:
-        metric_name = values.split(' ')[0]
-        metric_value = values.split(' ')[1]
-        print '{0},{1},{2}'.format(time_stamp, metric_name, metric_value)
+       if '{' in values:
+         data = re.sub(r"{[^{}]+}", lambda x: x.group(0).replace(",","_"), values)
+         data = re.sub(r"{[^{}]+}", lambda x: x.group(0).replace("{","_"), data)
+         data = re.sub(r"}","", data)
+         name = data.split(' ')[0]
+         value = data.split(' ')[1]
+         print '{0},{1},{2}'.format(time_stamp, name, value)
+       else:
+         metric_name = values.split(' ')[0]
+         metric_value = values.split(' ')[1]
+         print '{1},{0},{2}'.format(metric_name, time_stamp, metric_value)
```

Comment:
Consider removing above split, as we just need to add the timestamp to the lines.  And I think we might have a tool that already does that in the code base.